src/main/java/se/sundsvall/billingpreprocessor/api/InvoiceFilesResource.java

```diff
@@ -30,13 +32,23 @@ public InvoiceFilesResource(AsyncExecutorService service) {
 		this.service = service;
 	}
 
-	@PostMapping(produces = APPLICATION_PROBLEM_JSON_VALUE)
-	@Operation(summary = "Triggers creation of file entities from billing records flagged as APPROVED")
+	@PostMapping(path = "/create", produces = APPLICATION_PROBLEM_JSON_VALUE)
```

Comment:
It is usually discouraged to use verbs in REST-endpoints.

Maybe just a plain POST-request to the path "/invoicefiles/generation-batch" (or something similar to that)?

Reply:
I copied the solution in seab-loader and used 
```
/jobs/filecreator
/jobs/filetransferrer
```